### PR TITLE
Bump `Books.jl` to version 1.0

### PR DESCRIPTION
**Pull request by rikhuijzer**
Now using the `tufte-book` document class.

### CODE REVIEW

**Comment by rikhuijzer** [comment]
Slowly but surely getting there.

image

**Comment by rikhuijzer** [comment]
One of the issues is that URLs cannot be used in footnotes. Seems like an reasonably easy fix.

File: contents/why_julia.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ We reserved @sec:julia_wild for displaying successful Julia use cases in both ac
 **In Julia you can do your custom different stuff without loss of performance**.
 
 3. Had to debug code and somehow you see yourself reading a Fortran or C/C++ source code and having no idea what you are trying to accomplish?
-**In Julia you only read Julia code[^readable], no need to learn another language to make your original language fast**.
```
Comment (rikhuijzer) [inline contents/why_julia.md:56]: This footnote is referred to twice (also a bit lower on the page).

**Comment by rikhuijzer** [comment]
A very weird warning just came up inside `why_julia`:

```
warning: Can't begin an annotation when one is pending.
warning: Interpreting special command bann (pdf:) failed.
warning: >> at page="25" position="(341.172, 404.784)" (in PDF)
warning: >> xxx "pdf:bann<</Type/Annot/Subtype/Link/Border[0 0 0]/H/I/C[0 1 1..."
warning: >> Reading special command stopped around >><</Type/Annot/Subtype/Link/Border[0 0 0]/H/I/C[0 1 1]/A<</S/...<<
warning: Tried to end an annotation without starting one!
warning: Interpreting special command eann (pdf:) failed.
warning: >> at page="25" position="(188.611, 582.968)" (in PDF)
warning: >> xxx "pdf:eann"
```

I can't really see what is causing it nor any missing things in the text. So, let's hope it is nothing.

**Comment by rikhuijzer** [comment]
Yep. Let's stick to `Books.jl#main`, I've just committed a change which was only on my hard disk and not in the published branch for `v1.0` :laughing: It was a minor but nice/important change.

**Comment by storopoli** [comment]
ok tell me when to review it

File: contents/why_julia.md (hunk 1 above)
Comment (storopoli) [reply contents/why_julia.md:56]: Yes this is not a bug or error. It is intentional

**Review by storopoli — APPROVED** [review]

**Pull request merged**